### PR TITLE
Fix edit frequency

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes
Display frequency whenever the user clicks on Edit
frequency_edit

### CODE REVIEW

**Review by khaliqgant — CHANGES_REQUESTED** [review]
Thanks. The "every" should show when showing an existing frequency

"hour" by itself doesn't make sense.

Also just noticed this issue, "hours" is still accepted and is a confusing unit "every hours"


https://github.com/NangoHQ/nango/assets/1724137/5ba0fc5d-68d1-4565-b932-ca32436e76c6

File: packages/webapp/src/pages/Integration/FlowPage.tsx [hunk 1]
```diff
@@ -179,6 +184,11 @@ export default function FlowPage(props: FlowPageProps) {
         });
     };
 
+    const onCancelFrequncy = () => {
```
Comment (bodinsamuel) [inline packages/webapp/src/pages/Integration/FlowPage.tsx]:
```suggestion
    const onCancelFrequency = () => {
```

File: packages/webapp/src/pages/Integration/FlowPage.tsx [hunk 2]
```diff
@@ -326,18 +336,20 @@ export default function FlowPage(props: FlowPageProps) {
...
+                                                onKeyDown={(e) => {
                                                     if (e.key === 'Enter') {
                                                         onSaveFrequency();
                                                     }
                                                 }}
                                             />
                                             <XCircleIcon
                                                 className="flex h-5 w-5 text-red-400 cursor-pointer hover:text-red-700"
-                                                onClick={() => setShowFrequencyEditMenu(false)}
+                                                onClick={() => onCancelFrequncy()}
```
Comment (bodinsamuel) [inline packages/webapp/src/pages/Integration/FlowPage.tsx]:
```suggestion
                                                onClick={() => onCancelFrequency()}
```

**Review by bodinsamuel — APPROVED** [review]
Works great, minor comments

**Comment by hassan254-prog** [comment]
@khaliqgant , I have just updated my PR, to check for time units such as "every hours", "every days", "every mins" . It was also accepting "every mins" as an input which is a value less than 5 for minutes.

**Review by bodinsamuel — COMMENTED** [review]
I think the change requested by Khaliq were unrelated to this PR and it has introduced bugs unfortunately.
- Adding a space before `every` will add an other every
- Adding a space between `every` and `hours` make the cursor jump to the end
- Saving `every hours` complains about the missing unit and revert the input value

**Pull request closed without merging**